### DISCUSSION TITLE
wrong user id on PC prevents running 'vagrant up'

### DISCUSSION THREAD

**Discussion by LarsNauheimer** [chrisjackson-pellicle/HybPiper-RBGV#2] [General]
Hi Chris, 
In order to try the discussion feature I came up with an issue. :) 

I could run the vagrant up command from my linux machine, but got an error message when trying it from the windows PC. 
Apparently the user ID needs to be identical. 

```
λ vagrant up
The VirtualBox VM was created with a user that doesn't match the
current user running Vagrant. VirtualBox requires that the same user
be used to manage the VM that was created. Please re-run Vagrant with
that user. This is not a Vagrant issue.

The UID used to create the VM was: 1000
Your UID is: 0

λ id
uid=1263861(jc326597) gid=1049089 groups=1049089
```

Any ideas how to work around that?

**Comment by chrisjackson-pellicle** [comment]
Taking one for the team, Lars :)

That's odd, I didn't get this on my Windows machine. I'm not quite clear on how the VM could've been created with a different user, unless the project folder has been moved around, as described here: https://github.com/hashicorp/vagrant/issues/8630. Can you try the steps described there, or simply create a new folder with the vagrantfile in it and try `vagrant up` again? Let me know what happens...
